.github/workflows/main.yml

```diff
@@ -0,0 +1,17 @@
+name: Lighthouse CI
+on: push
+jobs:
+    lighthouseci:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+            - uses: actions/setup-node@v3
+              with:
+                  node-version: '18'
+                  cache: 'yarn'
+                  registry-url: 'https://registry.npmjs.org'
+            - run: npm install -g @lhci/cli@0.12.x
+            - name: Install dependencies
+              run: yarn --frozen-lockfile
+            - run: yarn docs:build
+            - run: lhci autorun
```

Comment:
Instead of running this here, I would use the yarn command `yarn lighthouse` so that you're using the locally installed version.